BrainCloudClient/Assets/BrainCloud/Client/BrainCloud/BrainCloudLobby.cs

```diff
@@ -468,12 +489,11 @@ private void pingNextItemToProcess()
             {
                 if (m_regionTargetsToProcess.Count > 0)
                 {
-                    for (int i = 0; i < NUM_PING_CALLS_IN_PARRALLEL && m_regionTargetsToProcess.Count > 0; ++i)

```

Comment:
Doing 2 pings on the same frame was causing issues with Unity's Ping system so I changed this to just do one ping at a time. The script only does 4 pings on a host and they usually return pretty quickly. If not, they should time out after 10 seconds. I don't think doing 2 pings at a time to resolve 4 pings is really a benefit.